Pages/Admin/Locations/Create.cshtml.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using ChemStoreWebApp.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc.Razor;
+using Microsoft.CodeAnalysis;
+using Location = ChemStoreWebApp.Models.Location;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Framework.Profiler;
+
+namespace ChemStoreWebApp.Pages.Admin.Location
+{
+    //[Authorize(Roles = "Admin")]
+    public class CreateModel : PageModel
+    {
+        private readonly ChemStoreWebApp.Models.chemstoreContext _context;
```

Comment:
Just to be a little cleaner, since there is already a using ChemStoreWebApp.Models, you should be able to just reference the chemstoreContext and Location classes

Reply:
Removed what I could, removing the `Models.` from `public Models.Location Location { get; set; }` causes an error so I am assuming it's necessary